README.md

```diff
@@ -178,6 +198,63 @@ make -k TOOLCHAIN=clang-18  CONFIG=Tsan  VERBOSE=1
 
 The makefile will use your system compiler, `c++`, if no toolchain name is provided, otherwise it will use the toolchain in the etc/ directory to perform the build. The Ninja multi config generator is used, with configurations for `RelWithDebugInfo`, `Debug`, `Tsan`, and `Asan` configured by default.
 
+#### Step by Step Build: Build and Run Tests
+
+CI current build and test flows:
+
+```shell
+# Configure build: default build production code + tests (BUILD_TESTING=ON by default).
+$ rm -rf .build/
+$ mkdir -p .build
+$ cd .build
```

Comment:
You can avoid changing the working directory to `.build` and back using CMake's `-B` parameter.

Reply:
Although starting deleting the build directory can fix a lot of strange problems, as CMake will only check some things the very first time its run in directory. `realclean` and not just the "clean" target can be necessary if you're changing anything significant. 
If I'm building with different toolchains I build in ./build/$toolchain, or ../build/$project/$toolchain, so as not to have crosstalk between incompatible builds.

Reply:
simplified with -B